include/stack/stack.h

```diff
@@ -0,0 +1,23 @@
+/*
+    AUTHORS: Siddharth Mahendraker, Shimi Smith
+*/
+
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#define MAX_SIZE 50
```

Comment:
`MAX_SIZE` and `DATA_SIZE` should only be defined once, in some shared header; otherwise, the two `MAX_SIZE` and `DATA_SIZE` variables should have different names.

Reply:
oh good point